docs/pages/apis/db-items.mdx

```diff
@@ -1,8 +1,149 @@
 import { Markdown } from '../../components/Page';
-import { ComingSoon } from '../../components/ComingSoon';
 
 # Database Items API
 
-<ComingSoon/>
+The database items API provides a programmatic API for running CRUD operations against the internal GraphQL resolvers in your system.
+Importantly, this API bypasses the Apollo server itself, instead invoking the resolver functions directly.
+The return values of this API are **internal item** objects, which are suitable to be returned from GraphQL resolvers.
+
+Refer to the [internal items guide](../guides/internal-items) for details on how to work with internal items in Keystone.
+
+This API executes the [`access control`](../guides/access-control) rules and [`hooks`](../guides/hooks) defined in your system.
+To bypass these, you can directly use the Prisma Client at [`context.prisma`](./context#database-access).
+
+For each list in your system the following API is available at `context.db.lists.<listName>`.
+
+```
+{
+  findOne({ where: { id } }),
+  findMany({ where, first, skip, sortBy }),
+  count({ where, first, skip }),
+  createOne({ data }),
+  createMany({ data }),
+  updateOne({ id, data }),
+  updateMany({ data }),
+  deleteOne({ id }),
+  deleteMany({ ids }),
+}
+```
+
+The arguments to these functions closely correspond to their equivalent GraphQL APIs, making it easy to switch between the programmatic API and the GraphQL API.
```

Comment:
This paragraph feels left over from the List Item API docs. The DB Item API is really not a thing you'd switch out directly for the GraphQL API, you could "between the List Items API and the DB Items API" maybe?